docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -1,63 +1,61 @@
 # Adds namespace to all resources.
-namespace: project-system
+namespace: project-v2-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: project-
+namePrefix: project-v2-
 
 # Labels to add to all resources and selectors.
 #commonLabels:
 #  someName: someValue
 
 bases:
-- ../crd
-- ../rbac
-- ../manager
+  - ../crd
+  - ../rbac
+  - ../manager
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
-- ../webhook
+#- ../webhook
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-- ../certmanager
+#- ../certmanager
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+#- ../prometheus
 
 patchesStrategicMerge:
-- manager_image_patch.yaml
   # Protect the /metrics endpoint by putting it behind auth.
-  # Only one of manager_auth_proxy_patch.yaml and
-  # manager_prometheus_metrics_patch.yaml should be enabled.
-- manager_auth_proxy_patch.yaml
-  # If you want your controller-manager to expose the /metrics
-  # endpoint w/o any authn/z, uncomment the following line and
-  # comment manager_auth_proxy_patch.yaml.
-  # Only one of manager_auth_proxy_patch.yaml and
-  # manager_prometheus_metrics_patch.yaml should be enabled.
-#- manager_prometheus_metrics_patch.yaml
+  # Only one of manager_auth_proxy_patch.yaml
+  - manager_auth_proxy_patch.yaml
+    # If you want your controller-manager to expose the /metrics
+    # endpoint w/o any authn/z, uncomment the following line and
+    # comment manager_auth_proxy_patch.yaml.
+  # Only one of manager_auth_proxy_patch.yaml
```

Comment:
Aren't the lines "Only one of manager_auth_proxy_patch.yaml" redundant now that there is only one option in this section? Also, to expose "/metrics" without auth, isn't it enough just to comment out "manager_auth_proxy_patch.yaml" line and perhaps the whole "patchesStrategicMerge"? In any case, the current comments do not seem correct as there is no "following line: to uncomment.

Reply:
I understand that the line `Only one of manager_auth_proxy_patch.yaml` clarifies that we cannot use more than one. I will give a look.

Reply:
I agreed with @draghuram. I guess we can drop these comments now.

> Also, to expose "/metrics" without auth, isn't it enough just to comment out "manager_auth_proxy_patch.yaml" line and perhaps the whole "patchesStrategicMerge"?

Yes

Reply:
CLosing  as solved